src/openassetio-core/hostApi/ManagerFactory.cpp

```diff
@@ -42,7 +42,7 @@ ManagerFactory::ManagerDetails ManagerFactory::availableManagers() const {
 
   // `HostSession` required for `settings()` calls.
   const managerApi::HostSessionPtr hostSession =
-      managerApi::HostSession::make(managerApi::Host::make(hostInterface_));
+      managerApi::HostSession::make(managerApi::Host::make(hostInterface_), logger_);
```

Comment:
[fix] Looks like was an oversight during development of #507, where we initially had `settings` as part of the `ManagerDetail`, and so required a `HostSession`.  We removed that but missed removing this line.

Reply:
Good spot, removed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/541/commits/03ce704860e1ac773242977a708bfc1b86a0b72f). Curious none of our linters pick up 'unused variable'.

Reply:
Good point.  I think the `make` functions should be marked `[[nodiscard]]`, I bet the linter(s) would complain then.

Reply:
Ahh yes, sorry I forgot that when we did it.

Reply:
@feltech  added https://github.com/OpenAssetIO/OpenAssetIO/issues/544